### ISSUE TITLE
Wrong Cell Type Detected

### ISSUE THREAD

**Issue by parth-ballu** [alanhay/html-exporter#25]
A cell having value '45 Pcs' is detecting as numeric type and ignoring 'Pcs' and giving only numeric value i.e. '45'. It happens only when a value is starting from a number.

**Comment by alanhay** [comment]
Yes, I can see that NumberFormat.getInstance().parse() which is used to check for numeric values is too lenient.

https://stackoverflow.com/questions/8416354/how-to-parse-numbers-more-strict-than-what-numberformat-does-in-java

So needs a stricter test.

As a workaround for now and if you have control of the HTML then explicitly marking it as a text cell should address the issue:

<td data-text-cell="true">13.54A</td>
